contracts/EtherTimelock.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.17;
+
+/**
+ * @title EtherTimelock
+ * @dev EtherTimelock is a contract that will allow a
+ * beneficiary to extract ether sent to it after a given release time
+ */
+contract EtherTimelock {
+    //beneficiary of funds
+    address public beneficiary;
+
+    //timestamp when funds are able to be released
+    uint64 public releaseTime;
+
+    function EtherTimelock(address _beneficiary, uint64 _releaseTime) {
+        require(_releaseTime > now);
```

Comment:
it shouldnt have a `require(_beneficiary != address(0))` here? to check that a valid address is used.

Reply:
sorry, are you saying it should have a require or that it should not?

Reply:
That it should, I believe

Reply:
Yes, is that it should :grimacing: